code/controllers/subsystem/shuttles.dm

```diff
@@ -1,4 +1,6 @@
 #define CALL_SHUTTLE_REASON_LENGTH 12
+/// Time elapsed from roundstart before shuttle calls are allowed
+#define SHUTTLE_REFUEL_DELAY 20 MINUTES
```

Comment:
Not `#undef`'d afterwards? (Also seems unnessecary to define it in the first place if it's only used for one variable.)